addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -775,6 +782,13 @@ blockquote {
                     }
                 }
             }
+            .dropdown-menu .dropdown-item-text.text-muted {
+                color: $text-muted !important;
+            
+                a {
+                    color: theme-color('primary');
+                }
+            }
```

Comment:
Why does this need to in o_ccX definition and not only o_cc ? :thinking: In any case, instead of `theme-color('primary')` it should probably be `$link-color`. Also, having elements with both dropdown-item-text and text-muted is clearly a bad practice (best to add a inner span with the text muted class inside the dropdown item), is there a way in stable to simply avoid that or not? (even with JS fixing the layout (if that is the rendering of search result, I suspect this is possible).

Reply:
@qsm-odoo Moved to o_cc and added a dedicated span. The template is indeed only rendered by JS, so the change should be ok for stable.
The style about the link will be added during the forward port to 15.0 which introduces the fuzzy search.
In saas-15.2, the dedicated span will also have to be added in enterprise's website_helpdesk.